data/human/human missions.txt

```diff
@@ -6849,3 +6849,320 @@ mission "FW Refugees to Humanika"
+
+	on visit
+		dialog `You arrive on <planet>, but realize that your escort carrying the exhibition installations has not arrived yet. Better depart and wait for your escorts to enter the system.`
+
+	on complete
+		payment 80000
+		event "adelita 2 offer expired" 1
+		dialog
+			`After navigating the thick fog of Moonshake's atmosphere, you find a motley crew waiting on your landing pad. Once you sign your landing documents with the Syndicate officials, one of crew walks up to you and hands you a note. It reads:`
+			`	"Meet me at the Common Grounds Cafe in the spaceport if you're interested in doing a project with us. Make sure you're not followed. - Adelita Zero"`
+
+event "adelita 2 offer expired"
+
+mission "Adelita 2"
+	description "Participate in an unauthorized filming of a Syndicate sweatshop on <destination>. (You will be paid <payment> after you complete the entire job.)"
+	name "Film Syndicate sweatshop on <planet>"
+	source "Moonshake"
+	destination "Maker"
+	passengers 3
+	"apparent payment" 200000
+	cargo "stealth drones" 1
+	to offer
+		has "Adelita 1: done"
+		not "adelita 2 offer expired"
+
+	on offer
+		conversation
+			`It takes a while to navigate through the bustling spaceport and the almost viscous fog of <origin>, but after finding an information screen, you're able to get to the Common Grounds Cafe. The cafe has a funky vibe to it and is quite busy. Nobody seems to recognize you, so you take a seat at a booth and order something to eat. Before your meal arrives, a woman, maybe in her late thirties with dark hair, sits down across from you and hails the server. She cheerfully orders a smoothie with a long list of ingredients, many of which you have never heard of, or can even pronounce.`
+			`	"Sorry for the cloak and dagger bit," she says to you finally. "We have enemies in high places. So Captain <last>, what did you think of the exhibition?"`
+			choice
+				`	"I found it quite moving."`
+				`	"It was OK."`
+					goto wasok
+				`	"It was too political for me."`
+					goto political
+			`	"I'm glad you liked it," she says, smiling.`
+				goto choice
+
+			label wasok
+			`	"Not exactly a thrilling review, but you're entitled to your opinion," she says, smiling.`
+				goto choice
+
+			label political
+			`	"Oh. Unfortunately, for many, everything is political. It was political when the Paradise Worlds tried to block the evacuation. And it was political when the Navy did it without the permission of Parliament. And the same goes for every single act of oppression and liberation since the dawn of humanity. Our oppressors are very political. Maybe you have the hope of being one some day; wealthy, authoritative. Or maybe you just don't care. Either way, you don't sound like a good fit for this project. But thanks for transporting the installation," she says with a smile. She gets her smoothie to-go and walks out.`
```

Comment:
```suggestion
			`	"Oh. Unfortunately, for many, everything is political. It was political when the Paradise Worlds tried to block the evacuation. And it was political when the Navy did it anyway without the permission of Parliament. And the same goes for every single act of oppression and liberation since the dawn of humanity. Our oppressors are very political. Maybe you have the hope of being one some day; wealthy, authoritative. Or maybe you just don't care. Either way, you don't sound like a good fit for this project. But thanks for transporting the installation," she says with a smile. She gets her smoothie to-go and walks out.`
```